modules/admin-home/rest/admin-config.php

```diff
@@ -86,7 +86,6 @@ private function get_elementor_editor_url( ?int $page_id, string $active_tab ):
 				'post' => $page_id,
 				'action' => 'elementor',
 				'active-tab' => $active_tab,
-				'active-document' => $active_kit_id,
```

Comment:
I have aligned our approach with the Elementor core plugin home screen. This approach is more stable.